src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkIslService.java

```diff
@@ -171,47 +169,32 @@ public void bfdPropertiesUpdate(IslReference reference) {
 
     /**
      * Process installed isl rule notification.
-     * @param reference isl reference
-     * @param payload response payload
      */
-    public void islDefaultRuleInstalled(IslReference reference, InstallIslDefaultRulesResult payload) {
+    public void islDefaultRuleInstalled(IslReference reference, Endpoint endpoint) {
         log.debug("ISL service received isl rule installed for {} (on {})", reference, reference.getSource());
         IslFsm islFsm = locateController(reference);
         IslFsmContext context;
-        if (payload.isSuccess()) {
-            context = IslFsmContext.builder(carrier, reference.getSource())
-                    .installedRulesEndpoint(Endpoint.of(payload.getSrcSwitch(), payload.getSrcPort()))
-                    .build();
-            controllerExecutor.fire(islFsm, IslFsmEvent.ISL_RULE_INSTALLED, context);
-        } else {
-            context = IslFsmContext.builder(carrier, reference.getSource())
-                    .build();
-            controllerExecutor.fire(islFsm, IslFsmEvent.ISL_RULE_TIMEOUT, context);
-        }
+        context = IslFsmContext.builder(carrier, reference.getSource())
+                .installedRulesEndpoint(endpoint)
```

Comment:
If you already use different events to report response, why for you need different fields in context to store them?